Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM broadinstitute/openidc-baseimage:1.8.5
+FROM broadinstitute/openidc-baseimage:1.8.6
```

Comment:
Since we're not using any of the authentication bits, maybe `FROM httpd:2.4` is more appropriate?